lib/readline.js

```diff
@@ -429,7 +428,9 @@ Interface.prototype._tabComplete = function() {
           }
         }
         handleGroup(self, group, width, maxColumns);
+      }
 
+      {
```

Comment:
For the time being we have been actively avoiding use of let due to
performance considerations, but it might not matter as much in REPL.

On Monday, August 1, 2016, Yorkie Liu <notifications@github.com> wrote:

> In lib/readline.js
> <https://github.com/nodejs/node/pull/7754#discussion_r73074208>:
>
> >
> > +      {
>
> let vs var shouldn’t matter
>
> I don't think so because here the brackets would create a scope with let
> declaration but not for var. For example:
>
> {
>   var f = completions.filter(function(e) { if (e) return e; });
>   var prefix = commonPrefix(f);
> }// here we actually can access `f` and `prefix`, this is not what we expect by brackets.// using `let` instead of `var` can make it expected result.
>
> Actually my suggestion is not removing brackets here and changing var ->
> let.
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/nodejs/node/pull/7754/files/37229c6bf7d911cadb1198a6c482ad489979a679#r73074208>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAa2eX6K1Q13s_9tot4zbaY0nFgE1vsmks5qboc2gaJpZM4JNz1q>
> .
>